test/unit/__snapshots__/AnimatedCounterDigit.test.js.snap

```diff
@@ -20,7 +20,7 @@ exports[`matches snapshot 1`] = `
     className="rollex-digit-lane-label"
     style={
       Object {
-        "height": 18,
+        "height": undefined,
```

Comment:
it signal a problem, probably `height` property has not been properly removed from source